### PR TITLE
docs: app mode and cluster mode explained

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR introduces changes to the introduction page. 

- Renamed to "What is Palette?"
- The index page now contains the Palette overview content
- Added a page dedicated to explaining app mode and cluster mode.
- The freemium page and credits page are combined

CleanShot 2022-12-14 at 11 41 37

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1041#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/01-introduction/05-palette-modes.md
Comment (ritawatson) [inline content/docs/01-introduction/05-palette-modes.md:44]:
```suggestion
Alternatively, you could use App Mode to offer your own Palette-managed host clusters as a PaaS experience to downstream consumers. This concept is easier explained through an example. Assume you are a system administrator, and you want to expose Kubernetes to various in-house development teams. You could deploy several Kubernetes clusters to various platforms and create a " development " cluster group. You also ensured every cluster is enabled for Palette Virtual Cluster by selecting the option before deployment. You can now direct your organization members to use App Mode and create Palette Virtual Clusters as needed, or you can create virtual clusters ahead of time for them. The organization members or downstream consumers can now focus on creating app profiles and deploying their applications. You have essentially enabled a Kubernetes PaaS experience for your organization.
```

File: content/docs/01-introduction/05-palette-modes.md
Comment (ritawatson) [inline content/docs/01-introduction/05-palette-modes.md:46]: This paragraph repeats and may not be needed. If you keep part or all of it, 'app mode' needs to be capitalized.

File: content/docs/01-introduction/05-palette-modes.md
Comment (ritawatson) [inline content/docs/01-introduction/05-palette-modes.md:51]:
```suggestion
You can quickly toggle between App Mode and Cluster Mode by navigating to the **User** menu at top right and selecting the mode you want.
```

File: content/docs/01-introduction/05-palette-modes.md
Comment (ritawatson) [inline content/docs/01-introduction/05-palette-modes.md:56]:
```suggestion
You might ask yourself, "How do I know which mode I should use?” The answer comes down to your objective. 

- Choose Cluster Mode if you want to enable Kubernetes capabilities for others or configure Palette. Cluster Mode, provides all the configuration options a power user wants. 
- Choose App Mode if you want to simply deploy an application using Kubernetes without the infrastructure overhead. If you want to simply try out Palette, App Mode is a good starting point. 

```

File: content/docs/01-introduction/05-palette-modes.md
Comment (ritawatson) [inline content/docs/01-introduction/05-palette-modes.md:58]:
```suggestion
App Mode may not meet your needs if your application requires a lot of resources. The Palette-managed cluster group, called Beehive, imposes a resource limitation that could prevent a resource-heavy application from launching successfully. Review the [Resource Quota](https://docs.spectrocloud.com/devx/resource-quota) documentation to understand App Mode limits.  If you already have Palette-managed Kubernetes host clusters deployed or available to you as a cluster group with Palette Virtual Clusters enabled, then leveraging App Mode is a great fit so you can focus on the developer experience aspect.
```

**Review by ritawatson — APPROVED** [review]
LGTM - with this comment:
-RBAC is usually role-based access control, but it sounds like Spectro Cloud has a unique use for it as resource-based access control. So readers don't think it's an error, I suggest italicizing resource in the spell-out. I didn't do that in my suggestion because I want to confirm my understanding.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1041#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1041#pullrequestreview-1218241761" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/01-introduction/05-palette-modes.md
Comment (karl-cardenas-coding) [reply content/docs/01-introduction/05-palette-modes.md:46]: I removed the first two sentences to reduce the repetition.

File: content/docs/01-introduction.md
Comment (karl-cardenas-coding) [reply content/docs/01-introduction.md]: Agree - we should break it out.

File: content/docs/01-introduction.md
Comment (karl-cardenas-coding) [inline content/docs/01-introduction.md:65]:
```suggestion
Learn more about Palette and how it can improve your Kubernetes experience and those in your organization. Try [Palette](https://console.spectrocloud.com/) for free today and experience a better way of working with Kubernetes. 
```

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | ec9318b475fcb22f9a0ddc5266560e83aa664509 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1041/librarium_build/1603460982749794304) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
